x-pack/test/functional_enterprise_search/apps/enterprise_search/with_host_configured/index.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ loadTestFile }: FtrProviderContext) {
+  describe('Enterprise Search', function () {
+    loadTestFile(require.resolve('./app_search/engines'));
+  });
+}
```

Comment:
So this test suite is only for local testing, or is there a follow-up plan to be able to run the suite against a running enterprise search instance on the CI?

Reply:
Correct, per the README `with_host_configured` only runs locally right now. Larry let us know that Spencer added the option to run a docker image as a sidecar service on CI, but currently we're leaning as a team more towards running separate Cypress E2E tests (since we already have a full suite of Cypress tests on ent-search). It's definitely on our list to follow up on!